src/olympia/amo/utils.py

```diff
@@ -673,8 +673,6 @@ def resize_image(source, destination, size=None, *, format='png', quality=80):
     When dealing with local files it's up to you to ensure that all directories
     exist leading up to the destination filename.
 
-    destination can be a string filename, or a file object.
```

Comment:
Maybe add that it should only be a filename.

Reply:
(merged so it's there for QA to test - I'll address comments in a follow-up commit tomorrow)

Reply:
so, this comment was only added in the previous PR #16831 and it's still kind of true in some cases - a `file` object (file-like-object) will work, but only a real file object that you don't care about being closed and where closing the file-like-object doesn't discard the data, like with `BytesIO`.  And only for jpeg format (`pngcrush_image` will fail for png files).  

Basically I realised my earlier comment was naive, and only applied for certain cases, and I either needed a) to write a longer explanation, and add some guard rails (`assert`, etc) or use different functions and contexts depending the type of the input, or b) just remove the comment so if someone was to try to reuse `resize_image` again in the future they would have to read and understand the code rather than rely on the comments.  We always use a filepath in the other uses anyway so it'd be a lot of work for an unused code path.